frontend/src/scenes/data-warehouse/editor/QueryWindow.tsx

```diff
@@ -72,7 +74,7 @@ export function QueryWindow({ onSetMonacoAndEditor }: QueryWindowProps): JSX.Ele
                     <span className="pl-2 text-xs">
                         {editingView && (
                             <>
-                                Editing {editingView.last_run_at ? 'materialized view' : 'view'} "{editingView.name}"
+                                Editing {isMaterializedView ? 'materialized view' : 'view'} "{editingView.name}"
```

Comment:
I swapped this out because I saw in our view list column we are using the status column to differentiate.